ts/src/test/static/request/kucoin.json

```diff
@@ -792,13 +792,34 @@
                 "input": []
             }
         ],
+        "fetchDepositAddress": [
+            {
+                "description": "default",
+                "method": "fetchDepositAddress",
+                "url": "https://api.kucoin.com/api/v3/deposit-addresses?currency=LTC",
+                "input": [
+                  "LTC"
+                ]
+            },
+            {
+                "description": "with networkCode",
+                "method": "fetchDepositAddress",
+                "url": "https://api.kucoin.com/api/v3/deposit-addresses?currency=USDT&chain=eth",
+                "input": [
+                  "USDT",
+                  {
+                    "networkCode": "ERC20"
```

Comment:
why `networkCode` instead of `network` ?

Reply:
i am used to use `networkCode` , but i've pushed a change to `network` atm.